enqueue.go

```diff
@@ -46,11 +46,9 @@ func Enqueue(queue string, class string, args []interface{}) (err error) {
 	}
 
 	// Push job in redis
-	err = conn.Send("RPUSH", fmt.Sprintf("%squeue:%s", namespace, queue), b)
-	if err != nil {
+	if err = conn.Cmd("RPUSH", fmt.Sprintf("%squeue:%s", namespace, queue), b).Err; err != nil {
```

Comment:
What I can say for now is:
1) It worked since it used redigo;
2) The tests are passing;
3) Apparently the arguments are converted to strings later: https://github.com/fzzy/radix/blob/master/redis/client.go#L151-L166